mobile/src/main/java/org/openhab/habdroid/core/NotificationHelper.kt

```diff
@@ -94,6 +95,19 @@ class NotificationHelper(private val context: Context) {
         )
     }
 
+    private fun createActionIntent(action: CloudNotificationAction, notificationId: Int): PendingIntent {
+        val intent = Intent(context, NotificationActionReceiver::class.java)
+        intent.putExtra(NOTIFICATION_ID_EXTRA, notificationId)
+        intent.putExtra(NOTIFICATION_ACTION_LABEL, action.label)
```

Comment:
`action` is parcelable, so I'd suggest to use it as extra directly instead of splitting it in parts and reconstructing it later.